components/mpas-ocean/bld/namelist_files/namelist_definition_mpaso.xml

```diff
@@ -717,7 +717,7 @@ Default: Defined in namelist_defaults.xml
 
 <entry id="config_cvmix_background_scheme" type="char*1024"
 	category="cvmix" group="cvmix">
-Scheme for background diffusivity, 'constant' for constant with depth and space, 'BryanLewis' for vertically variable, 'none' for no background diffusivity
+Scheme for background diffusivity, 'constant' for constant with depth and space, 'BryanLewis' for vertically variable, 'LatDep' for latitude dependent, 'none' for no background diffusivity
```

Comment:
please add 'LatDep' in the 'Valid values' line below.

Reply:
Good catch, fixed.  Although it doesn't seem to affect the ability to set it in the user_nl file which I thought is what the "valid values" was supposed to trap.  Maybe another problem in the implementation of the scripts?

Reply:
From what I remember in the past, I got an error on an invalid entry at runtime, not at setup. So, I am thinking the 'Valid values' are only for documentation.
At least that's my recollection.